### ISSUE TITLE
Open Source Post taken Down.

### ISSUE THREAD

**Issue by Morningstar88** [Morningstar88/kalki-search#337]
1111

**Comment by Morningstar88** [comment]
![image](https://github.com/user-attachments/assets/cc13eaf8-0fba-4e96-8354-531577b859ce)

![image](https://github.com/user-attachments/assets/3f83a4db-5b56-4953-8847-f194eea915f8)

**Comment by Morningstar88** [comment]
![image](https://github.com/user-attachments/assets/0084e974-a463-4758-95f8-32f8b987311d)

![image](https://github.com/user-attachments/assets/2a4f6948-6654-4149-85df-f45b79780e2d)

![image](https://github.com/user-attachments/assets/5379aa96-a960-401a-af63-1aaa8d069a10)

**Comment by Morningstar88** [comment]
[TheITMan19](https://www.reddit.com/user/TheITMan19/)
•
[24m ago](https://www.reddit.com/r/opensource/comments/1etvj5b/comment/lig84em/)
Are you ok? :)

**Comment by Morningstar88** [comment]
I think you want sharepoint.

EDIT: also, i think there is a language barrier here. If english is not your first language then that would explain the disconnect i got from reading but your responses to other people seemed to enforce you being ridiculous. So if it's a language thing then I apologize for my assumptions. git still isnt a good fit for this (or writing, sharepoint or something similar if you care about doc versions)

If English is your first language, then you can continue thinking i am an asshole...

*

[JonnyRocks](https://www.reddit.com/user/JonnyRocks/)

[mArKoLeW](https://www.reddit.com/user/mArKoLeW/)
•
[3m ago](https://www.reddit.com/r/opensource/comments/1etvj5b/comment/ligbyoy/)
I like this analogy more: GitHub and git is like pornhub and porn. The hubs are there to manage, vote and do a lot more with git and porn. But you can use git without GitHub the same way you can watch porn without pornhub.

[mArKoLeW](https://www.reddit.com/user/mArKoLeW/)

Jonny Rocks.

**Comment by Morningstar88** [comment]
github is a site for hosting git repos. github is not git. just a website. github is to git as amazon is to books. git is for version control on source code. You might want something like sharepoint.

*

I appreciate you taking the time to explain that to me so slowly and patiently. So, Java and Javascript are the same, right? Feel free to use any analogies you think are relevant.

Answer:

I think you are misunderstanding my use case, and the situation here.

A family has a coconut farm. The older generation can't use computers.

The son has learned some wordpress at school.

He can be taught GFM and issue raising within 15-20 minutes, as no code is involved.

The can now make a simple GFM Readme to advertise the coconut farm.

They can use the forking feature to set up versions of the farm in different provinces, where the climate and local situation are different. They now have 5 forks / repos for five farms.

The presence on the internet allows them to connect with local engineers.

Other entrepreneurs fork the original doc, and set up farms quickly... the original Repo owner now becomes a coconut farm consultant.

So far we are only using Github. No code. But Version Control, forking, and repos are immensely helpful to people here, for many reasons.

Later, to add real code, you could transition into a real website, ghpages, a live site on vercel, codepens/fiddles. The Github API could be used to monitor, display and advertise local projects on a separate site. Then, NodeJS could be brought in. A list of relevant terminal commands might give an intro to the server.

Starting with a repo offers a smooth transition to real backend coding.

Then, git itself could be used to update multiple projects at the same time. I wonder if there's a command that can update several repos at once. You could also use git checkout to look at versions of the farm/project in the past. git bisect could even be used to find pain points and historical mistakes during the farming process. Maybe rebase could be used to set up a new version of the project overnight.

git itself could also be used in microfinance networks. Im still thinking how.

Github actually allow and encourage Github for writers. They are not elitist.

Your assumptions, estimation of my ability, and response, are monumentally patronizing. This is why people stay away from techies. In future please ask for clarification first. TIA.

**Comment by Morningstar88** [comment]
![image](https://github.com/user-attachments/assets/05feadfc-a0ff-46ad-bb32-43fb77ddc426)

[cantaprete](https://www.reddit.com/user/cantaprete/)
•
[37m ago](https://www.reddit.com/r/opensource/comments/1etvj5b/comment/lig5k0c/)
PitHub, the alternative for peach farms.

BAN

mydnic
Jonny Rocks

**Comment by Morningstar88** [comment]
![image](https://github.com/user-attachments/assets/6581973e-9ab3-49ce-8360-f909380f914c)

**Comment by Morningstar88** [comment]
Do Github and Version Control have Real World Applications way beyond Software Development?
I'm in South East Asia now...

What about showing someone how to manage a Repo? No code. Just GFM and issue raising. I figure that is 15-30 minutes onboarding time, for anyone who can type and use FB/email.

There are issues in Asia here with water purification, construction, farming, traffic and more.

One example: Traffic Control... a huge problem in Bali.

A repo could be used to monitor a particular traffic surge spot.

You could also help people fund and start coconut farms in their area. Scale them locally, using a separate Repo for each farm to account for different towns/climates.

A project starter, with a few dozen forks, would become consultants in their niche.

Almost any project could be scaled, monitored and localized using the fork feature, which is transcendental for project management.

Crowdfunding projects that you could fork might work. GH has funding features now, and can be coupled with Open Collective and other crowdfunding sites.

I predict the version control industry may be bigger in a few years... with "Repo Maintainer" a pragmatic path into other coding jobs, and tech companies.

Many more options.

tl;dr, Github can be reinvented like Jimi Hendrix changed the guitar.

Does anyone have any ideas?

Sorry, this post has been removed by the moderators of r/opensource.

**Comment by Morningstar88** [comment]
I_eat_Limes_
2:53 AM
Hi, thanks for removing that post. I understand that my writing style was a but rushed, but I clarified and edited as best I could...

Im taking this straight to [/r/subredditdrama](https://www.reddit.com/r/subredditdrama).

With respect, I think the answers from your community, and the hasty removal, are atrociously condescending and censorious.

You have given a very poor account of yourselves, IMO. Not even giving me a chance to clarify.

I shall see you in the drama court, gentlemen!

**Comment by Morningstar88** [comment]
![image](https://github.com/user-attachments/assets/3fd4e0f4-8e1f-4dc0-bd96-2fa30d104c64)

**Comment by Morningstar88** [comment]
![image](https://github.com/user-attachments/assets/000cf522-b9f5-45cb-be5f-1820d2355bbf)

**Comment by Morningstar88** [comment]
![image](https://github.com/user-attachments/assets/922320e6-d82f-4c53-be66-b1a42d4e0613)
